src/libs/nav.js

```diff
@@ -87,7 +87,7 @@ export const TitleManager = () => {
   const newTitle = Utils.cond(
     [_.isFunction(title), () => title({ ...params, queryParams: query })],
     [title, () => title],
-    getAppName
+    () => getAppName({ capitalInitial: true })
```

Comment:
This is to capitalize `t` in `the` in the `RARE-X` site name (i.e. The RARE-X Data Analysis Platform) displayed on the site's browser tab.